client/app/components/home/home.css

```diff
@@ -142,3 +142,15 @@ ul.menu li:hover {
 #create-folder-errors {
     min-height: 17px;
 }
+
+#page-header {
+    position: fixed;
+    width: 100%;
+    top: 0;
+    left: 0;
+    z-index: 5;
```

Comment:
@yegor86 
5 is quite enough :) There is no limitation for z-index in CSS http://stackoverflow.com/a/491105